static/cms/admin/config.yml

```diff
@@ -235,5 +235,63 @@ collections:
           - { label: 'Maltese', value: 'mt' }
           - { label: 'Welsh', value: 'cy' }
           - { label: 'Belarusian', value: 'be' }
+  - name: community-events
+    label: Community Events
+    label_singular: Community Event
+    folder: src/collections/community-events
+    slug: '{{title}}'
+    create: true
+    fields:
+      - name: title
+        label: Title
+      - name: host
+        label: Host
+        widget: relation
+        collection: community-event-hosts
+        search_fields: [name]
+        value_field: name
+        display_fields: [name]
+      - name: url
+        label: URL
+      - name: category
+        label: Category
+        widget: select
+        options:
+          - { label: 'Tournament', value: 'tournament' }
+          - { label: 'Social', value: 'social' }
+          - { label: 'Learning', value: 'learning' }
+          - { label: 'Generic', value: 'generic' }
+      - name: startDateTime
+        label: Start Date/Time
+        widget: datetime
+        default:
+        format: YYYY-MM-DDTHH:mm:ssZ
+      - name: useStartTime
+        label: Use Start Time
+        widget: boolean
+        default: true
```

Comment:
Do events usually have a start time? If so `default: true` would make sense to me.

Reply:
I think events usually do have a start time, however, I've reconsidered this to be `default: false` as I'd rather a user forget to toggle this to `true` and have no time versus they forget to toggle it `false` and have some random default time.